common/lib/xmodule/xmodule/html_module.py

```diff
@@ -98,7 +98,7 @@ class HtmlModuleMixin(HtmlBlock, XModule):
         'js': [
             resource_string(__name__, 'js/src/collapsible.js'),
             resource_string(__name__, 'js/src/html/imageModal.js'),
-            resource_string(__name__, 'js/common_static/js/vendor/draggabilly.pkgd.js'),
+            resource_string(__name__, 'js/common_static/js/vendor/draggabilly.js'),
```

Comment:
I find the usage of draggabilly by the HTML module to be very mysterious. @clrux, do you know how the HTML module uses it? Have you verified that the functionality still works (if it ever did)?